js/settings.js

```diff
@@ -70,30 +70,38 @@ const settingsObject = {
       'jqueryui/autoGrowInput': 'assets/jqueryui/autoGrowInput',
     }
   },
-  paths: {
+  paths: {    
+    "datatables.net-buttons": "assets/jquery.dataTables.buttons.min",
```

Comment:
Why was the reordering done? Seems for me like only unnecessary diff

Reply:
the order here is important

Reply:
Why? If there exist some rules for ordering, why weren't they described?

Reply:
It's because of the cross-dependencies. X requires Y, Y requires jquery.
To be honest, this exact order has been found by guessing